src/easystar.js

```diff
@@ -431,6 +449,23 @@ EasyStar.js = function() {
         return false;
     };
 
+    /**
+     * -1, -1 | 0, -1  | 1, -1
+     * -1,  0 | SOURCE | 1,  0
+     * -1,  1 | 0,  1  | 1,  1
+     */
+    var calculateDirection = function (diffX, diffY) {
+        if (diffX === 0, diffY === -1) return 'bottom'
+        else if (diffX === 1, diffY === -1) return 'bottom-left'
```

Comment:
I get it, but it might be handy for users who don't like having magic strings all over their codebase. Nobody would be forced to use them, but they would be there if someone wanted to. Lets include them.